raster-foundry/raster-foundry
# Implement project-layer annotation UI

## Overview

This PR moves our annotation functionality into the project layers UI. There are a few changes made for UI consistency.

### Checklist


### Demo

![image](https://user-images.githubusercontent.com/2442245/53115396-1afe4400-3514-11e9-8317-580bbf2dedbd.png)


### Notes

I will make an issue for adding annotation stats to the project layer stats endpoint.


## Testing Instructions

 * Make annotations using the layer UI for at least 2 layers. Make sure that they annotations do not cross over to the wrong layer, etc.

* Make annotations using the old UI. Ensure this still works as expected

Closes #4535
